gtsfm/utils/geometry_comparisons.py

```diff
@@ -205,3 +205,50 @@ def compute_relative_unit_translation_angle(U_1: Optional[Unit3], U_2: Optional[
     angle_rad = np.arccos(dot_product)
     angle_deg = np.rad2deg(angle_rad)
     return angle_deg
+
+
+def compute_translation_to_direction_angle(
+    i2Ui1: Optional[Unit3],
+    wTi2: Optional[Pose3],
+    wTi1: Optional[Pose3]
+) -> Optional[float]:
```

Comment:
I have a function which does similar stuff in the front-end. I compute angles between two `Unit3`s. I think that is a cleaner API instead of having a `Unit3` and two `Pose3`s.

Reply:
Indeed, and I call that function here. I've been seeing this as a utility file, and `compute_translation_to_direction_angle` is a high-level utility function, I could move the logic of finding the unit translation between two translations to within the function that calls this, but would that not make the code less modular?

Reply:
IMHO it is ok to have this as a separate more modular function